net-proxy/tayga/tayga-0.9.2-r4.ebuild

```diff
@@ -24,6 +24,9 @@ src_prepare() {
 	default
 	sed -e '/^CFLAGS/d' \
 		-i configure.ac || die "sed failed"
+	# Workaround for bug #939535:
+	# eautoreconf does not run autoheader with --force
+	rm config.h.in
```

Comment:
`|| die`

Reply:
https://devmanual.gentoo.org/ebuild-writing/error-handling/

You are correct it will fail either way

However, Gentoo policy is that "external commands can fail due to out of memory situations which prevent the system from having enough resources to call fork()", and that in such a scenario it is important to fail immediately rather than later on, because a later error could be confusing.

I don't make the rules. :)

Reply:
I see. Done